### ISSUE TITLE
Apply button in Font dialog

### ISSUE THREAD

**Issue by mxa** [pure-data/pure-data#1286]
Can we have an Apply button in the font dialog too?
(So we don't have to go back to the menu every time to bring back the window until the size fits)

**Comment by sebshader** [comment]
@mxa what do you mean? Are you talking about Edit>Font? In the font dialog here (on osx) the font size automatically changes when different sizes are chosen in the radio.

If you're talking about the "stretch" parameter then there's #1248

**Comment by mxa** [comment]
Yes, exactly. Applying the stretch. I find myself trying to guess how much i should stretch... 110%? OK, no not enough, mousing to get the dialog again, then another 120%, oops, too much, mousing again. ...

**Comment by sebshader** [comment]
@mxa you can input a stretch factor and then click on the current font-size radio button to "apply" right now
hopefully the PR I mentioned will make the interface more intuitive..

**Comment by mxa** [comment]
Nice workaround tho. Thanks.
